tests/unit/components/offer-banner/OfferBanner.spec.jsx

```diff
@@ -13,12 +13,12 @@ const buttonActions = [
 const mockOffer = {
   _id: 'id',
   authorAvgRating: 4.3,
```

Comment:
we don`t have this field anymore either, so you can remove it

Reply:
fixed